### DISCUSSION TITLE
about protobuf_c_boolean

### DISCUSSION THREAD

**Discussion by zhuguang11111** [protobuf-c/protobuf-c#644] [Q&A]
hello, is anyone know the function of protobuf_c_boolean type?when will it generated and what does it for ?there is less introduction about this in the doc. could anyone can help explain it? thanks a lot!

**Comment by sWAGrOGER** [comment]
Well a boolean is like a checkbox so you could do a if (c_boolean) and (!c_boolean) not looked it this project but a boolean is basically a trigger.
